frontend/src/theme/selectStyle.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from "./colors";
+
+const optionBackgroundColor = (isSelected: boolean, isFocused: boolean) => {
+  if (isSelected) {
+    return colors.teal[100];
+  }
+  if (isFocused) {
+    return colors.teal[50];
+  }
+  return colors.white.default;
+};
+
+const selectStyle: any = {
+  singleValue: (provided: any) => ({
+    ...provided,
+    color: colors.gray[500],
+    borderColor: colors.gray[200],
+    boxShadow: "none",
+  }),
+  control: (provided: any, state: any) => ({
+    ...provided,
+    backgroundColor: state.isDisabled ? colors.teal[50] : colors.white.default,
+    borderColor: colors.gray[200],
+    boxShadow: "none",
+    borderRadius: "4px",
+    "&:hover": {
+      border: `1px solid ${colors.gray[200]}`,
```

Comment:
it was kind of hard to track down what should happen when you hover, but I think the new color should be teal.300. 
```suggestion
      border: `1px solid ${colors.teal[300]}`,
```